app/controllers/insured/verification_documents_controller.rb

```diff
@@ -34,6 +38,9 @@ def upload
   end
 
   def download
+    consumer_role = person_consumer_role
+    authorize consumer_role, :verification_document_download?
```

Comment:
@saipraveen18 You are already assigning `@person.consumer_role` to `@consumer_role`. You can use it directly like the one below.

```suggestion
    authorize @consumer_role, :verification_document_upload?
```

Reply:
Updated